src/antsibull_docs/process_docs.py

```diff
@@ -40,6 +43,78 @@ def get_collection_namespaces(collection_names: Iterable[str]) -> dict[str, list
     return namespaces
 
 
+_CAPITAL_LETTERS = tuple("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
```

Comment:
See `string.ascii_uppercase` from the stdlib

Reply:
f3f1f25360de562bb5f027ed339a70b3021ca505